web/massastation/src/layouts/LayoutStation/LayoutStation.tsx

```diff
@@ -71,13 +72,11 @@ export function LayoutStation({ ...props }) {
 
   useEffect(() => {
     if (isSuccess) {
-      plugins.forEach((plugin) => {
-        if (plugin.name === 'Massa Wallet') {
-          setPluginWalletIsInstalled(true);
-        }
-      });
+      setPluginWalletIsInstalled(
+        plugins.some((plugin) => plugin.name === 'Massa Wallet'),
```

Comment:
I guess we are using that 3 times already. It would be nice to have a source of truth on that name. Could you please just add that under a const MASSA_WALLET under /const folder?